build.sbt

```diff
@@ -262,6 +262,14 @@ lazy val testkit =
           .map(_ => file)
           .get
       },
+      DocTest / scalacOptions ++= {
+        scalaBinaryVersion.value match {
+          case "2.13" =>
+            Seq("-Wconf:cat=scala3-migration:warning")
```

Comment:
This is unnecessary if we add more explicit types. should I remove this option and add explicit types?

```
[error] my-slick-dir-path/doc/code/LiftedEmbedding.scala:104:13: under -Xsource:3, inferred slick.lifted.ProvenShape[(String, Int, Double, Int, Int)] instead of Nothing [quickfixable]
[error] Scala 3 migration messages are errors under -Xsource:3. Use -Wconf / @nowarn to filter them or add -Xmigration to demote them to warnings.
[error] Applicable -Wconf / @nowarn filters for this fatal warning: msg=<part of the message>, cat=scala3-migration, site=com.typesafe.slick.docs.LiftedEmbedding.main.Coffees.*
[error]         def * = ???
[error]             ^
[error] my-slick-dir-path/doc/code/LiftedEmbedding.scala:582:15: under -Xsource:3, inferred Any instead of Pair[Any,Any] [quickfixable]
[error] Scala 3 migration messages are errors under -Xsource:3. Use -Wconf / @nowarn to filter them or add -Xmigration to demote them to warnings.
[error] Applicable -Wconf / @nowarn filters for this fatal warning: msg=<part of the message>, cat=scala3-migration, site=com.typesafe.slick.docs.LiftedEmbedding.main.PairShape.buildValue
[error]           def buildValue(elems: IndexedSeq[Any]) = Pair(elems(0), elems(1))
[error]               ^
[error] my-slick-dir-path/doc/code/LiftedEmbedding.scala:583:15: under -Xsource:3, inferred slick.lifted.Shape[Level, _, _, _] instead of PairShape[Level,Nothing,U,Nothing] [quickfixable]
[error] Scala 3 migration messages are errors under -Xsource:3. Use -Wconf / @nowarn to filter them or add -Xmigration to demote them to warnings.
[error] Applicable -Wconf / @nowarn filters for this fatal warning: msg=<part of the message>, cat=scala3-migration, site=com.typesafe.slick.docs.LiftedEmbedding.main.PairShape.copy
[error]           def copy(shapes: Seq[Shape[_ <: ShapeLevel, _, _, _]]) = new PairShape(shapes)
[error]               ^
[error] three errors found
[error] (testkit / Doctest / compileIncremental) Compilation failed
```